CHANGELOG.md

```diff
@@ -70,7 +70,160 @@ changes since the last release, see the [diff on GitHub][unreleased].
     for the `v3.0.0.0-alpha.5` release. Leave the release links under the release section.
 -->
 
+### Added
+
+- Added the [`--what-if` (`-w`)][ur-aa] option to the [dsc config set][cmd-cset] command. When you
+  call `dsc config set` with the `--what-if` option, DSC doesn't actually invoke the resources to
+  enforce the desired state. Instead, it returns the expected output for the command, showing the
+  before and after state for each resource instance.
+
+  The output for the `dsc config set` operation with the `--what-if` operation is the same as an
+  [actual configuration set operation][ur-ab], except that the metadata field
+  [executionType][ur-ac] is set to `WhatIf` instead of `Actual`.
+
+  In this release, the generated output is synthetic, based on the results of the resources' `test`
+  operation. In the future, resources will be able to participate in what-if operations, reporting
+  more specifically how they will change the system. For example, participating resources could
+  indicate whether an actual set operation will require a reboot or updating the `PATH`
```

Comment:
Maybe a better example than `PATH` is whether the current user has the correct permissions to apply some configuration